nimble/host/include/host/ble_gap.h

```diff
@@ -1582,6 +1582,19 @@ struct ble_gap_ext_adv_params {
 
     /** Advertising Set ID */
     uint8_t sid;
+
+    /** Version of HCI command to be used
+     *  0 : No version set, choose V2 or fallback to v1
+     *  1 : V1
+     *  2 : V2
+     */
+    unsigned int version:2;
```

Comment:
this is not needed - you can return an error if phy options are non-zero and v2 is not supported, otherwise send either v1 or v2.

Reply:
Addressed. Please check once